migrations/versions/77a3ca142468_.py

```diff
@@ -0,0 +1,42 @@
+"""empty message
+
+Revision ID: 77a3ca142468
+Revises: a983b8b1865d
+Create Date: 2022-06-17 11:38:00.557432
+
+"""
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.dialects import postgresql
+
+# revision identifiers, used by Alembic.
+revision = '77a3ca142468'
+down_revision = 'a983b8b1865d'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.alter_column('rh_accounts', 'account',
+               existing_type=sa.TEXT(),
+               nullable=True)
+    op.drop_constraint('rh_accounts_account_key', 'rh_accounts', type_='unique')
+    op.drop_constraint('uq_rh_accounts_org_id', 'rh_accounts', type_='unique')
+    op.create_unique_constraint('uq_rh_accounts', 'rh_accounts', ['account', 'org_id'])
+    op.drop_constraint('rh_accounts_account_check', 'rh_accounts', type_='check')
+    op.create_check_constraint('ck_rh_accounts', 'rh_accounts', 'NOT(account IS NULL OR org_id IS NULL)') 
```

Comment:
As you have set `account` column as nullable, do we still need to consider `account` column inside condition?

Reply:
oh that code was auto-generated. I had already removed 🤔.

Reply:
@r14chandra - so what is expected condition here?

Reply:
account and org_id should not be null.

Reply:
> As you have set `account` column as nullable, do we still need to consider `account` column inside condition?

It is set as not null as earlier. I just removed `nullable=false` and added the check contraint for both the columns.